client/src/app/project-form/project-metadata-schema.json

```diff
@@ -260,6 +260,17 @@
       "type": "array",
       "user_friendly": "INSDC study accession"
     },
+    "ega_accessions": {
+      "description": "A list of European Genome phenome Archive dataset or study accessions.",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "EGA[DS][0-9]{11}"
+      },
+      "example": "EGAS00000000001; EGAD00000000002",
+      "guidelines": "Enter any EGA study or dataset accessions that relate to the project. Should start with EGAD or EGAS, study accession preferred.",
+      "user_friendly": "EGA Study/Dataset Accession(s)"
+    },
```

Comment:
I think we could just copy the whole JSON from here so that the id will also be updated https://schema.humancellatlas.org/type/project/14.2.0/project